### ISSUE TITLE
USE_PROMETHEUS flag

### ISSUE THREAD

**Issue by holmesb** [astr0n8t/tasmota-power-exporter#65]
Hi,

Please be patient with a newb question. I've seen that Tasmoto firmware has a [USE_PROMETHEUS](https://github.com/arendst/Tasmota/blob/66d69e09c92d741d9728654ecfef63572fbbdc48/tasmota/include/tasmota_configurations.h#L832) flag. Does this obviate? In other words, this project avoids the necessity to compile Tasmoto with USE_PROMETHEUS, which is a non-default flag, and apply such custom-built Tasmoto firmware to all devices? 

Just trying to understand the circumstances to use tasmota-power-exporter.

Many thanks.

**Comment by astr0n8t** [comment]
Yes that's correct.  If it's compiled with that flag I don't think you need to worry about this and can just use Prometheus with it correctly.
